src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.scss

```diff
@@ -160,4 +176,42 @@ h3 {
 
 ::ng-deep .mat-select-panel.allow-long-text {
   max-width: unset;
+}
+
+mat-slide-toggle {
+  position: absolute;
+  left: 0em;
+  top: 0em;
+}
+
+::ng-deep {
+
+  & .mat-tooltip {
+    font-size: 12px;
+  }
+
+  & .mat-slide-toggle-label {
+    padding: 1em;
+  }
+
+  & .mat-slide-toggle-thumb {
+    position: relative !important;
```

Comment:
Don't like all of the "!importants". By giving the toggle a css class and using that to target it (as suggested in the above comment) it shouldn't need them.  If it still does we can work out additional steps, but "!important" should be avoided, apart from when fighting someone else's crap "!important" css :-)